### DISCUSSION TITLE
Ender 3 v2 with Micro-swiss direct drive, all metal hot-end and BLTouch. X and Y homing position off build plate

### DISCUSSION THREAD

**Discussion by Conn308** [mriscoc/Ender3V2S1#1106] [General]
so im trying to print TPU with my ender, I was able to flash the firmware and screen to the new professional version from an older jyers version. ever since i flashed to the new version, when i home the X and Y positions, they stop just a few millimeters off the edge of the build plate ( the test line tries to print off the side of the build plate).... i have a 4.2.2 board with a DWIN screen, Micro-swiss direct drive, all metal hot-end and a bltouch

i reset the physical dimensions of the plate along with the X and Y min and max, saved and restarted printer

any help would be greatly appreciated!

**Comment by mriscoc** [comment]
Check https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides
